lib/tasks/asset_sync.rake

```diff
@@ -10,21 +9,4 @@ namespace :assets do
       AssetSync.clean
     end
   end
-
-end
-
-if Rake::Task.task_defined?("assets:precompile:nondigest")
```

Comment:
Ah, I was wrong, and I'm ok with that!

I had to generate a custom initializer via `rails g asset_sync:install --provider=AWS` and then edit it to include a line
```
config.include_webpacker_assets = true
```
When I ran `rake assets:sync`, I got my "packs" directory, and the Webpacker asset was inside.

🎉